src/controllers/skheras.js

```diff
@@ -0,0 +1,81 @@
+'use strict';
+import Skhera from '../models/skhera';
+
+export const getByUser = async ({params: { id }}, res) => {
+  try {
+    const skheras = await Skhera.find({author: id });
+    return res.status(200).send(skheras);
+  } catch (error) {
+    return res.status(500).send(error.message)
+  }
+}
+
+export const getById = async (req, res) => {
+  try {
+    const skheras = await Skhera.findById(req.params.id);
+    return res.status(200).send(skheras);
+  } catch (error) {
+    return res.status(400).send(error.message);
+  }
+}
+
+export const getEstimatedPrice = async  ({ body: {distance, to, deliver}}, res) => {
```

Comment:
I know the application did not specify how to calculate the price for the delivery process but When faced with choices like these you should try and think of solutions and discuss them with your team.. Delivery Cost is right at the core of the application, it's a crucial part of the Business plan and should be given a lot of thought .
Now your solution is not flexible at all, values are hard coded and the if statement is flawed (it's basically if >100 {} else{} the else if clauses do not get executed ever.

Reply:
yes is true. it's one temporary solution, i have one preposition but i not had the time to explore and discuss them.